holmes/plugins/toolsets/infrainsights/infrainsights_client.py

```
@@ -0,0 +1,205 @@
+"""
+InfraInsights Client for HolmesGPT Toolsets
+
+This client handles authentication, instance discovery, and credential management
+for the InfraInsights multi-instance architecture.
+"""
+
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)